static/app/views/performance/utils.tsx

```diff
@@ -232,16 +232,19 @@ export function trendsTargetRoute({
   } else {
     modifiedConditions.setFilterValues('tpm()', ['>0.01']);
   }
-  if (conditions.hasFilter('transaction.duration')) {
-    modifiedConditions.setFilterValues(
-      'transaction.duration',
-      conditions.getFilterValues('transaction.duration')
-    );
-  } else {
-    modifiedConditions.setFilterValues('transaction.duration', [
-      '>0',
-      `<${DEFAULT_MAX_DURATION}`,
-    ]);
+  // Metrics don't support duration filters
+  if (!organization.features.includes('performance-new-trends')) {
+    if (conditions.hasFilter('transaction.duration')) {
+      modifiedConditions.setFilterValues(
+        'transaction.duration',
+        conditions.getFilterValues('transaction.duration')
+      );
+    } else {
+      modifiedConditions.setFilterValues('transaction.duration', [
+        '>0',
+        `<${DEFAULT_MAX_DURATION}`,
+      ]);
+    }
```

Comment:
We have a UI based smoothing function `smoothTrend` which uses ASAP for high feature detection, we use it on the trends display in summary but never ported it back to trends. It might let you pick a higher interval and would also hide the outliers that the max duration filter was previously covering. In the long term we might need to lean on the histogram metrics implementation w/ it's built-in outliers but for now that might make it look better? For a future PR, this is fine atm.

Reply:
oh okay, good to know 👍